packaging/datadog-agent/source/setup_agent.sh

```diff
@@ -11,10 +11,10 @@ set -u
 # SCRIPT KNOBS
 #######################################################################
 # Update for new releases, will pull this tag in the repo
-DEFAULT_AGENT_VERSION="5.32.7"
+DEFAULT_AGENT_VERSION="7.25.1"
```

Comment:
This script only supports the v5 release line of the Agent, so this v7 version will not work with this script